manager/get_api_gateway_id.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import boto3
+import os
+
+
+def get_api_gateway_id():
+    cluster_name = os.environ["CORTEX_CLUSTER_NAME"]
```

Comment:
Since there are only 2 variables, it may be easier to follow the code that calls this python file if these values are accepted as args

Reply:
(see comment above)